### PR TITLE
Refactor [Spiget Steam StackExchange static] badge services

### PR DESCRIPTION
**Pull request by RDIL**
Ref https://github.com/badges/shields/issues/5555

### CODE REVIEW

File: services/spiget/spiget-download-size.service.js [hunk 1]
```diff
@@ -3,9 +3,7 @@
 const { BaseSpigetService, documentation, keywords } = require('./spiget-base')
 
 module.exports = class SpigetDownloadSize extends BaseSpigetService {
-  static get category() {
-    return 'size'
-  }
+  static category = 'size'
 
   static get route() {
```
Comment (jabbar86) [inline services/spiget/spiget-download-size.service.js:8]: @RDIL Convert this also to static property

**Review by jabbar86 — CHANGES_REQUESTED** [review]
```
static route = {
base: 'spiget/download-size',
pattern: ':resourceId',
}
```

File: services/spiget/spiget-download-size.service.js (hunk 1 above)
Comment (RDIL) [reply services/spiget/spiget-download-size.service.js:8]: Whoops, missed that.

**Review by calebcartwright — DISMISSED** [review]

**Comment by calebcartwright** [comment]
@RDIL - not sure what was going on with Circle earlier but after closing and reopening the checks were finally run, and the main job is failing due to formatting issues. Could you run `npm run prettier` and then push the fix to get the CI checks passing?

**Comment by calebcartwright** [comment]
I wonder if Circle isn't running the checks automatically because your fork isn't coming from your user profile :thinking:

**Comment by RDIL** [comment]
Could be.  CircleCI hasn't really liked me in the past on other repositories as well.

**Review by calebcartwright — APPROVED** [review]
Thanks!

**Pull request merged**